components/OrderDetails/PrintShare.js

```diff
@@ -0,0 +1,53 @@
+import { View, StyleSheet } from "react-native";
+import * as Print from "expo-print";
+import { shareAsync } from "expo-sharing";
+import AntDesign from "@expo/vector-icons/AntDesign";
+import html_template from "../../utils/html_template";
+import useGetServicesHistory from "./hooks/useGetServicesHistory";
+import useStatusHistory from "./hooks/useStatusHistory";
+
+const PrintShare = ({ service }) => {
+  const { serviceHistory } = useGetServicesHistory(service);
+  const orderStatuses = useStatusHistory(service);
+
+  const print = async () => {
+    const htmlContent = html_template(service, serviceHistory, orderStatuses);
+    await Print.printAsync({ html: htmlContent });
+  };
+
+  const printToFile = async () => {
+    const htmlContent = html_template(service, serviceHistory, orderStatuses);
+    const { uri } = await Print.printToFileAsync({ html: htmlContent });
+    await shareAsync(uri, { UTI: ".pdf", mimeType: "application/pdf" });
+  };
+
+  return (
+    <View style={styles.container}>
+      <AntDesign name="printer" size={28} color="black" onPress={print} style={styles.icon} />
```

Comment:
Ček ovdi imamo jednu ikonu za print a drugu printToFile? Jel to kao saveToFile?

Reply:
Mislim da bi se ova druga mogla zvat shareAsPdf?

Reply:
done